includes/WikiManager.php

```diff
@@ -381,4 +407,32 @@ private function recacheJson( $wiki = null ) {
 		$cWJ->resetDatabaseList();
 		$cWJ->update();
 	}
+
+	public function getUserRequestCount( $user, int $viewLevel, string $status ) {
```

Comment:
Should probably be implemented more like:
```php
	public function getUserRequestCount( $user, int $viewLevel, string $status ) {
		$conditions = [
			'cw_user' => $user->getId(),
			"cw_visibility <= $viewLevel"
		];
		if ( $status != '*' ) {
			$conditions['cw_status'] = $status;
		}

		$count = $this->cwdb->newSelectQueryBuilder()
			->select( 'cw_visibility' )
			->from( 'cw_requests' )
			->where( $conditions )
			->caller( __METHOD__ )
			->fetchRowCount();

		return $count;
	}
```
This resolves the issues of `$viewLevel` being unused, and `$this->getAuthority` being an undefined method.